services/IBF-pipeline/pipeline/lib/pipeline/rainfalldata.py

```diff
@@ -261,21 +259,21 @@ def findTrigger(self):
                                 how='left', op='intersects')
 
         # interpolate NaN cells
-        known = ~np.isnan(compare_gdf['forecast_time'])
+        known = ~np.isnan(compare_gdf['leadTime'])
```

Comment:
@jannisvisser We should move these constant strings `leadTime` and `triggerLevel` to a separate constants file. Similar to how we have enums / models in TypeScript.